editor/script/find_in_files.cpp

```diff
@@ -1358,17 +1350,13 @@ void FindInFilesContainer::_notification(int p_what) {
 				const int margin_right = -bottom_panel_style->get_margin(SIDE_RIGHT);
 				const int margin_bottom = -bottom_panel_style->get_margin(SIDE_BOTTOM);
 
-				if (get_theme_constant("margin_top") != margin_top) {
+				if (get_theme_constant("margin_top") != margin_top && get_theme_constant("margin_left") != margin_left && get_theme_constant("margin_right") != margin_right && get_theme_constant("margin_bottom") != margin_bottom) {
```

Comment:
This condition is unnecessary tbh. Fetching all these constants isn't much cheaper than just overwriting them.